laba4.py

```diff
@@ -26,21 +26,23 @@ def get_name(self):
         return self.__name
     
     def set_carats(self, carats):
-        self.__carats = carats
+        self.carats = carats
         
     def set_price(self, price):
         self.__price = price
         
     def set_name(self, name):
         self.__name = name
     
-    def __str__(self):
-        return (f"Коштовний камінь: {self.__name}, {self.__carats} карат, "
+    def __str(self):
```

Comment:
just like with a destructor, __str must be __str__, beacuse you overwritting python metod